eZ/Publish/API/Repository/Tests/ContentServiceAuthorizationTest.php

```diff
@@ -584,6 +585,60 @@ public function testDeleteContentThrowsUnauthorizedException()
         $this->contentService->deleteContent($contentInfo);
     }
 
+    /**
+     * Test for the deleteContent() method.
+     *
+     * @see \eZ\Publish\API\Repository\ContentService::deleteContent()
+     */
+    public function testDeleteContentThrowsUnauthorizedExceptionWithLanguageLimitation()
```

Comment:
Maybe it would be good to:
```suggestion
    public function testDeleteContentThrowsUnauthorizedExceptionWithLanguageLimitation(): void
```
for all methods